tareas/RuizGarcia/Tarea3Diego.jl

```diff
@@ -0,0 +1,719 @@
+# una forma analítica de probar la intuición, opte por visualizar el conjunto de Cantor.
+
+#-
+function cantor(n,limits)
+    cantor = Vector{Float64}[[limits[1],limits[2]]]
+
+    for k in 1:n
+        cantoraux = Vector{Float64}[]
+        for el in cantor
+            xi = el[1]
+            xf = el[2]
+            d = (xf-xi)/3
+            new1 = [xi,xi+d]
+            new2 = [xf-d,xf]
+            push!(cantoraux,new1,new2)
+        end
+        cantor = copy(cantoraux)
+    end
+    return cantor
+end
+
+
+
+function plt_cantor(n,limits)
+    vec = cantor(n,limits)
+    
+    plt=plot(vec[1],[1.2,1.2],color="purple",lw=2,alpha=0.7,xlabel="",ylabel="",label="")
+    xlims!(limits)
+    ylims!((0,2))
+    
+    for k in 2:length(vec)
+        vek = vec[k]
+        plot!(vec[k],[1.2,1.2],color="purple",lw=3.3,alpha=0.7,label="")
+    end
+    display(plt)
+end
+
+#-
+## Pruebo el funcionamiento
+cantor(2,(0,1))
+
+#-
+## Hago la gráfica.
+cantor = plt_cantor(5,(-p₊,p₊))
+savefig(cantor,"cantor.png")
+
+#-
+# ![Fig 17](cantor.png "Fig. 17")
+
+
+#-
+# Desgraciadamente, mi gráfica del conjunto de cantor no se ve tan similar a los puntos iniciales de la figura 16. 
+# No estoy seguro sobre cómo proceder.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+#-
+# ## Ejercicio 4:
+
+# NOTA: Para este ejercicio es útil que tengan instaladas las paqueterías
+# `Images.jl`, `Colors.jl` y `ColorSchemes.jl`, u otras. Sugiero que las instalen
+# en la tarea (usando `Pkg.add(...)`) sin guardar los cambios en `Project.toml`
+# (es decir, no hagan `git add Project.toml` y `git commit Project.toml` en ningún momento).
+
+# Consideren el mapeo $z_{n+1} = z_n^2 + c$ y la condición inicial $z_0 = 0$,
+# donde $z_n$ y $c$ son números complejos, y donde variaremos $c$ como número complejo.
+# Construiremos el conjunto de Mandelbrot.
+
+# - Primero, muestren que si para alguna $n$ $|z_n|^2 > 4$, entonces la evolución
+# de la órbita del mapeo diverge, es decir, $|z_n|$ va a crecer indefinidamente.
+
+# - Escriban una función que devuelve el número de iteración de $z_0$, dada $c$, en la que
+# diverge, es decir, el valor de $n$ cuando $|z_n|^2>4$. Si la condición inicial *no*
+# ha divergido hasta el número de pasos máximo (`pasos_max`, que se fija inicialmente),
+# la función devolverá `pasos_max+1`. Llamaremos a esta función `numero_pasos`.
+
+# - En el plano complejo ($\textrm{Re}(c)$, $\textrm{Im}(c)$) usaremos un código
+# de color para representar el número de iterados en que la condición inicial $z_0$
+# escapa al variar $c$. Esto es, identificaremos con un color (distinto) la primer $n$ tal que
+# $|z_n|^2 > 4$, que se obtiene con la función `numero_pasos`; si llegamos al número
+# de pasos máximo (`pasos_max`), usaremos el color negro. (Fuera de la región de
+# interés también podemos usar el negro.)
+
+# ### Respuesta:
+
+## Podemos mostrar que los valores absolutos no convergen con un ejemplo numérico.
+using Distributions
+
```

Comment:
```suggestion
Qc(x,c) = x^2+c
```
Agrega esto para que no de error el codigo al correrlo seguido